frappe/docs/user/en/guides/basics/client-side-api.md

```diff
@@ -0,0 +1,84 @@
+Web applications are based on Client-Server architecture. In Client-Server architecture communication takes place in form of request response calls.
+
```

Comment:
This opening is too generic, the person reading the document probably knows this!

A better opening would be:

```
`frappe.call` calls an AJAX request on the specified white listed server method.

This can be used while scripting forms or building custom UI on the Desk. Sample use cases are:

- Querying data from the server using REST API
- Calling whitelisted server methods

### Documentation

[share the function call]

### Example
```